src/hub/filters.py

```diff
@@ -1,55 +1,12 @@
-from datetime import timedelta
-
 from django.contrib.postgres.search import TrigramSimilarity
-from django.db.models import Count, F, FileField, Q
-from django.utils import timezone
+from django.db.models import FileField
 from django_filters import rest_framework as filters
 
-from discussion.reaction_models import Vote
-
 from .models import Hub
 
 
-class ScoreOrderingFilter(filters.OrderingFilter):
```

Comment:
Did this filter get moved, or did we remove it entirely?

Reply:
Was testing something and forgot to put it back in.